db/migrate/20161128193435_add_researcher_quote_to_project_contents.rb

```diff
@@ -0,0 +1,7 @@
+class AddResearcherQuoteToProjectContents < ActiveRecord::Migration
+  def change
+    add_column :project_contents, :researcher_quote, :text
```

Comment:
If that's the case, what needs to be done to backfill the data? Can they be left nil or do they need to be backfilled with a blank-string default? If so, can that be done in the migration, or is that too heavy for the db?